styles/globals.css

```diff
@@ -149,10 +154,33 @@
     @apply text-default-text underline;
   }
 
+ 
+
   .is-sticky {
     position: fixed;
     z-index: 999;
     top: 0;
     animation: 500ms ease-in-out 0s normal none 1 running fadeInDown;
   }
+  
+  #applicationTitle{
+    @apply sm:w-8/12
+  }
+
+  #mainHeader > header > div {
+    padding-left: 0px !important;
+    padding-right: 0px !important;
+  }
+
+  #mainHeader > header > div.ds-container > nav > ul > li {
+    margin-left : 0px !important;
```

Comment:
It looks like these CSS changes apply to the Design System header. We should not be styling DS components, this is fragile and not the best way of doing things. If the DS component needs styling, we should open a PR with them, so that the fix applies to all users of the DS, and there is no risk of the style breaking as things are updated.

Reply:
Sure, I will revert this change and create a PR on design-system repo